src/request/export/Export.css

```
@@ -42,25 +42,21 @@ ul.nq-list li > .nq-icon {
     padding-bottom: 0;
 }
 
-.page#recovery-words .nq-text {
-    text-align: center;
-    color: var(--nimiq-orange); /* Can be removed when CSS specificity is fixed in nimiq-styles */
-    margin-bottom: 0;
-}
-
 .page#recovery-words .recovery-words {
-    max-height: 100%;
+    height: 39rem;
```

this seems too large of a number to me. Looks better with 
```suggestion
    height: 35rem;
```
combined with the following margins and paddings in my opinion. 
Not really a requested change, just my personal preference.